.github/pull_request_template.md

```diff
@@ -1,7 +1,8 @@
 ## PR Checklist
 
 * [ ] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.
-* [ ] *if updating templates, it is helpful to post a screenshot of what the changes look like.*
+* [ ] *post a screenshot of templates and/or gallery changes*
 * [ ] *if updating templates, ensure that your parameters and steps have meaningful names.*
-* [ ] *if adding new templates, ensure that your parameters and columns have display names.*
-* [ ] *if adding new templates, or changing items in a gallery, it's helpful to post a screenshot of what the gallery looks like now*
+* [ ] *if updating templates, grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds*
+* [ ] *if updating templates, remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook.*
+* [ ] *if adding new templates, ensure that your parameters and columns have display names.*
```

Comment:
parameters having meaningful names was up earlier, should this be merged into that one?

* if adding or updating templates, ensure all steps have meaningful names
* if adding or updating templates, ensure all parameters and grid columns have display names set so they can be localized
* ensure that parameters do not have duplicate id values or they will fail PR validation (parameter ids are used for localization)
* ensure that steps do not have duplicate names or they will fail PR validation (step names are used for localization)